runtime/layouts/apollo3.ld

```diff
@@ -1,9 +1,14 @@
 /* Layout for the Apollo3 MCU, used by the examples in this repository. */
 
-MEMORY {
-  FLASH (X) : ORIGIN = 0x00040000, LENGTH = 0x00060000
-  RAM   (W) : ORIGIN = 0x10002000, LENGTH = 0x2000
-}
-
 TBF_HEADER_SIZE = 0x60;
+
+FLASH_START = 0x00040000;
+FLASH_LENGTH = 0x00060000;
+
+/* Note that the SRAM address may need to be changed depending on
+ * the kernel binary, check for the actual address of APP_MEMORY!
+ */
```

Comment:
Doesn't this comment apply to `FLASH_START` and `FLASH_LENGTH` as well? All of these files are fundamentally kernel-specific and there's no way we can change that :-(

Reply:
FWIW, as part of these changes I just copied this comment from one layout file to all of them, to make things consistent (as it technically applies to all boards equally). I can revert that part of the changes, or remove the comment everywhere, if that makes it less confusing.

Reply:
To the extent possible, I'm trying to keep everything (including documentation) in `runtime/libtock_layout.ld` to avoid tons of copy+pasting.

Reply:
Makes sense, I'll remove this comment then -- it mostly serves as a hint to developers that app loading might break depending on kernel layout expectations. As you said, this applies to flash in a similar section, and there doesn't seem to be a real benefit of putting it in the `libtock_layout.ld` file, as this is not going to be a file developers look at to debug why their applications aren't loading. The kernel as a pretty solid process loading debug infrastructure, which might be sufficient at this point.